tools/ci_build/github/azure-pipelines/nuget/templates/gpu.yml

```diff
@@ -266,3 +266,11 @@ jobs:
       command: 'push'
       searchPatternPush: '$(Build.BinariesDirectory)/nuget-artifact/final-package/Microsoft.ML.OnnxRuntime.Gpu.$(NuGetPackageVersionNumber).nupkg'
       feedPublish: 'OnnxRuntime'
+
```

Comment:
We need to push the "Managed" nuget as well. This package takes a dependency on the Managed package. Please see- https://github.com/microsoft/onnxruntime/blob/ef4d73e88772bd99d1d35d05dc917e0a39906fe2/tools/ci_build/github/azure-pipelines/nuget/templates/cpu.yml#L277. (I guess this applies to the Internal Nuget feed as well). When consuming this Nuget, it will try to resolve all dependencies and won't find the Managed nuget.

Reply:
@snnn as FYI as #3816 seems to have missed uploading the "Managed" package generated from this pipeline to the internal feed.

Reply:
Should CPU/GPU share the same "Managed" package?

Reply:
They should eventually but currently they do not. The "dev" packages have a timestamp in the version and thus the CPU and GPU native packages depend on "virtually" different Managed packages even if the contents are the same. We should eventually tackle this "unification" problem. But for now, we should upload Managed packages from each pipeline. This won't be a problem for Release packages as the version doesn't contain a timestamp in them.

Reply:
For example, the CPU native "dev" package with version native-t1 will depend on Managed-t1. The GPU native package will depend on Managed-t2. Uploading Managed-t1 and Managed-t2 to the same feed is possible. When someone tries to consume native-t1 they will get Managed-t1 and so forth.

Reply:
Thank you for explaining this. 
@ryanlai2 , would you please follow his advice?

Reply:
Yes, I will try to follow how the CPU pipeline pushes the "Managed" nuget. I will need some help taking a look.